inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/group/InlongGroupServiceImpl.java

```diff
@@ -170,6 +173,29 @@ public String save(InlongGroupRequest request, String operator) {
         return groupId;
     }
 
+    @Override
+    @Transactional(rollbackFor = Throwable.class)
+    public String save(InlongGroupRequest request, UserInfo opInfo) {
```

Comment:
Can we reuse the old method since most of the codes are identical?

Reply:
Yes, business logic and page are currently consistent

This idea is to separate the page from the open api first, and implement the call logic separately. After completing the subsequent implementation, we will see if they are the same in the end: if they are consistent, we will merge them into one; if they are inconsistent, we will merge them into one. deal with separately